src/LookupModel.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $lookup)
         $this->latitude = $this->getOrNull('latitude', $lookup);
         $this->longitude = $this->getOrNull('longitude', $lookup);
         $timezone = $this->getOrNull('timezone', $lookup);
-        if (!is_null($timezone)) {
+        if (!is_null($timezone) && in_array($timezone, timezone_identifiers_list())) {
```

Comment:
So it's literally hardcoded right now to return `n/a` if a timezone can't be determined.
That said, I think the `$timezone !=== 'n/a'` should have some performance benefits over `in_array`. So I think my vote would be for @jdpedrie's suggestion.